modules/tensor_mechanics/doc/content/modules/tensor_mechanics/beam_vandv.md

```diff
@@ -0,0 +1,254 @@
+
+!listing euler_pipe_axial_force.i
+
+### Results
+
+The analytical displacement at the end is given by:
+\begin{equation}
+\Delta = \frac {PL}{AE} = 3.537 \times 10^{-3} in
+\end{equation}
+
+The displacement at the end obtained from the MOOSE using 10 elements is 3.537$\times$10$^-$$^3$ in as shown in [fig:euler_pipe_axial_force].
+
+!media media/tensor_mechanics/vandv/euler_pipe_axial_force2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_pipe_axial_force
+       caption=Displacement of the Euler beam under axial load.
+
+## Large strain/large rotation of cantilever beam
+
+A 1D cantilever beam of 4 m is modeled using beam elements in MOOSE. The beam has a cross section area of 1 ${m}^2$, moment of inertia about Y and Z axis of 0.16 ${m}^4$, Young's modulus of elasticity of 1$\times$10$^4$ N/${m}^2$, poisson's ratio of -0.99, shear modulus of 1$\times$10$^8$ N/${m}^2$ and shear coefficient of 1. A point load of magnitude 300 N is applied at the free end of the cantilever beam in Y-direction.
+
+!listing euler_finite_rot_y.i
+
+### Results
+
+For this beam, the dimensionless parameter alpha is given by:
+\begin{equation}
+alpha = \frac {kAGL^2}{EI} = 1\times 10^6
+\end{equation}
+ Since, the value of alpha is quite high, the beam behaves like a thin beam where shear effects are not significant. The analytical solution for the displacements due to large rotations and large strain of the cantilever beam in X and Y direction is -1 m and 2.4 m respectively ([!citet](bishopanddrucker1945)).
+
+The displacement at the free end of the cantilever beam obtained from the MOOSE  is -0.954 m in X direction as shown in [fig:euler_finite_rot_x] and 2.37 m in Y direction as shown in [fig:euler_finite_rot_y].
+
+
+!media media/tensor_mechanics/vandv/euler_finite_rot_x2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_finite_rot_x
+       caption=Displacement of the cantilever beam with large strain and rotation in X direction
+
+
+!media media/tensor_mechanics/vandv/euler_finite_rot_y2.png
+       style=width:450px;margin-left:110px;float:center;
+       id=fig:euler_finite_rot_y
+       caption=Displacement of the cantilever beam with large strain and rotation in Y direction
+
+
+## Torsion
+
+A 1D cantilever beam of 1 m is modeled using beam elements in MOOSE. The beam has a cross section area of 0.5 ${m}^2$, moment of inertia about Y and Z axis of 1$\times$10$^-$$^5$ ${m}^4$, Young's modulus of elasticity of 2$\times$10$^9$ N/${m}^2$, poisson's ratio of 0.3 and shear coefficient of 1. A torsion of magnitude 5 N/m is applied at the free end of the cantilever beam.
```

Comment:
You can write the negative exponents within curly braces, such as `1$\times$10$^{-5}$`, but what has been written here renders fine